src/pages/Register.tsx

```diff
@@ -0,0 +1,215 @@
+            toast.error('Registration failed. Check the server.');
+        }
+    };
+
+
+    return (
+        <Container
+            fluid
+            className={`d-flex justify-content-center align-items-center ${Object.keys(errors).length > 0 ? 'mt-5' : 'mt-4'
+                } mb-4`}
+        >
+            <Row className="w-100">
+                {/* Left Section */}
+                <Col lg={6} className="d-none d-lg-flex justify-content-center align-items-center">
+                    <img
+                        src="signup.jpeg"
+                        alt="Logo"
+                        className="img-fluid"
+                        style={{
+                            maxHeight: '80%',
+                            maxWidth: '90%',
+                            marginLeft: '10%',
+                        }}
+                    />
+                </Col>
+
+                {/* Right Section */}
+                <Col
+                    lg={6}
+                    xs={12}
+                    className="d-flex justify-content-center align-items-center mt-4 mb-4"
+                >
+                    <div
+                        className="p-4 shadow rounded position-relative card-with-corners rounded-4"
+                        style={{
+                            maxWidth: '100%',
+                            marginLeft: 'auto',
+                            marginRight: 'auto',
+                        }}
+                    >
+                        <Form onSubmit={handleSubmit(onSubmit)}>
+                            <h4 className="text-center mt-2">Sign Up</h4>
+                            {/* Social Icons */}
+                            <div className="d-flex justify-content-center gap-3 my-3">
+                                <Button variant="outline-primary" className="rounded-circle p-2">
+                                    <svg
+                                        xmlns="http://www.w3.org/2000/svg"
+                                        width="24"
+                                        height="24"
+                                        fill="currentColor"
+                                        className="bi bi-google"
+                                        viewBox="0 0 16 16"
+                                    >
+                                        <path d="M15.545 6.558a9.4 9.4 0 0 1 .139 1.626c0 2.434-.87 4.492-2.384 5.885h.002C11.978 15.292 10.158 16 8 16A8 8 0 1 1 8 0a7.7 7.7 0 0 1 5.352 2.082l-2.284 2.284A4.35 4.35 0 0 0 8 3.166c-2.087 0-3.86 1.408-4.492 3.304a4.8 4.8 0 0 0 0 3.063h.003c.635 1.893 2.405 3.301 4.492 3.301 1.078 0 2.004-.276 2.722-.764h-.003a3.7 3.7 0 0 0 1.599-2.431H8v-3.0" />
+                                    </svg>
+                                </Button>
+                                <Button variant="outline-primary" className="rounded-circle p-2">
+                                    <svg
```

Comment:
Move this svg to a separate file and name icon-github.svg and place it under src/assets/icons